README.md

```diff
@@ -156,6 +155,18 @@ kubelogin convert-kubeconfig -l msi --client-id msi-client-id
 kubectl get no
 ```
 
+#### Azure CLI token login (non interactive)

```

Comment:
suggestion: add a note saying it's for AKS only or provide instruction how to setup for customer's own AAD app.

Reply:
As I replied above - happy to add this in, I just didn't know it would be required. Do you have a pointer to doc on what would need to be added? If not, I'll try to figure it out Monday.

Reply:
Note added to the description.